src/devices/Display/samples/Large4Digit14SegmentDisplay/Program.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Display;
+
+const string SupportedCharacters = "0123456789aAbBcCdDeEfFgGhHiIjJkKlLmMnNoOpPqQrRsStTuUvVwWxXyYzZ-=_|°[]     ";
+
+// Initialize display
+using Large4Digit14SegmentDisplay display = new(I2cDevice.Create(new I2cConnectionSettings(busId: 1, Ht16k33.DefaultI2cAddress)))
+    {
+        // Set max brightness
+        Brightness = Ht16k33.MaxBrightness
+    };
+
+// Write "Iot" on the display
+display.Write("Iot");
+
+// Wait 2 seconds
+Thread.Sleep(2000);
+
+// Change 'o' to '°' in "IoT"
+display[1] = (Segment14)FontHelper.GetCharacter('°');
+
+// Wait .5 seconds
+Thread.Sleep(500);
+
+// Write 42 to the left side of the display
+display.Write(42, Alignment.Left);
+
+// Set blinkrate to once per second
+display.BlinkRate = BlinkRate.Blink1Hz;
+
+// Wait 5 seconds
+Thread.Sleep(5000);
+
+// Set blinkrate to twice per second
+display.BlinkRate = BlinkRate.Blink2Hz;
+
+// Write 42 to the right side of the display
+display.Write(42, Alignment.Right);
+
+// Set brightness to half max
+display.Brightness = Ht16k33.MaxBrightness / 2;
+
+// Wait 2 seconds
+Thread.Sleep(2000);
+
+// Turn off blinking
+display.BlinkRate = BlinkRate.Off;
+
+// Write time to the display
+display.Write(DateTime.Now.ToString("Hmm").PadLeft(4));
+
+// Wait 3 seconds
+Thread.Sleep(3000);
+
+// Turn on buffering
+display.BufferingEnabled = true;
+
+// Write -42°C to display using "decimal point" between 3rd and 4th digit as the ° character
+display.Write("-42C");
```

Comment:
Shouldn't this use `-42.C` as string input for the described effect using the dot?

Reply:
It should. I still have a bit of tidying up to do with these decimals places I think.

Reply:
Done